asapdiscovery-data/asapdiscovery/data/tests/test_target_schema_v2.py

```diff
@@ -24,15 +25,10 @@ def sars2_spruced_pdb():
     return pdb
 
 
-@pytest.mark.parametrize("ttype", ["sars2_mpro", "mers_mpro", "sars2_mac1"])
-def test_targettype_init(ttype):
-    tt = TargetType(ttype)
-    assert tt.value == ttype
-
-
-def test_targettype_init_bad_name():
-    with pytest.raises(ValueError):
-        _ = TargetType("bad_name")
+@pytest.fixture(scope="session")
+def oedu():
```

Comment:
might be good to change this to `oedu_fn` or something so that it's clear this is a file and not an actual `OEDesignUnit` object

Reply:
good idea.